diff --git a/wrapper/target.cpp b/wrapper/target.cpp
index 82bf65c..a2520e2 100644
--- a/wrapper/target.cpp
+++ b/wrapper/target.cpp
@@ -741,6 +741,9 @@ bool Target::setup() {
           (stdlib == StdLib::libstdcxx && usegcclibs)) {
         fargs.push_back("-nostdinc++");
         fargs.push_back("-Qunused-arguments");
+        if ((SDKOSNum >= OSVersion(11, 1)) && (stdlib == StdLib::libcxx)) {
+          fargs.push_back("-lc++");
+        }
       }

       if (stdlib == StdLib::libstdcxx && usegcclibs && targetarch.size() < 2 &&

diff --git a/build_clang.sh b/build_clang.sh
index 9cf7da9d4..4a866be13 100755
--- a/build_clang.sh
+++ b/build_clang.sh
@@ -184,7 +184,7 @@ cmake ../llvm \
   -DCMAKE_INSTALL_PREFIX="$INSTALLPREFIX" \
   -DCMAKE_BUILD_TYPE=Release \
   -DLLVM_ENABLE_ASSERTIONS=OFF \
-  -DLLVM_ENABLE_PROJECTS="clang;libcxx;libcxxabi" \
+  -DLLVM_ENABLE_PROJECTS="clang" \
   -DLLVM_TEMPORARILY_ALLOW_OLD_TOOLCHAIN=1
 $MAKE -j $JOBS VERBOSE=1
 